File: Java/Lista_03(Estrutura_Repetitiva)/Par_Impar.java
import java.util.Scanner;

public class Par_Impar {

	public static void main(String[] args) {

		Scanner s = new Scanner(System.in);
		
		System.out.print("Quantos numeros voce vai digitar? ");
		int n = s.nextInt();
		
		for(int i=0;i<n;i++) {
			System.out.print("Digite um numero: ");
			int x = s.nextInt();
			
			if(x%2==0 && x>0) {
				System.out.println("PAR POSITIVO");
			}
			else if(x%2==0 && x<0) {
				System.out.println("PAR NEGATIVO");
			}
			else if(x%2!=0 && x>0) {
				System.out.println("IMPAR POSITIVO");
			}
			else if(x%2!=0 && x<0) {
				System.out.println("IMPAR NEGATIVO");
			}
			else {
				System.out.println("NULO");
			}
		}
		
		s.close();
		
	}

}

File: C++/Lista_03(Estrutura_Repetitiva)/Par_Impar.cpp
#include <iostream>

using namespace std;

int main(){

    int n, x;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> n;

    for(int i=0;i<n;i++){
        cout << "Digite um numero: ";
        cin >> x;
        if(x>0 && x%2==0){
            cout << "PAR POSITIVO" << endl;
        }
        else if(x>0 && x%2!=0){
            cout << "IMPAR POSITIVO" << endl;
        }
        else if(x<0 && x%2==0){
            cout << "PAR NEGATIVO" << endl;
        }
        else if(x<0 && x%2!=0){
            cout << "IMPAR NEGATIVO" << endl;
        }
        else{
            cout << "NULO" << endl;
        }
    }

    return 0;

}